services/slam/builtin/cartographer_int_test.go

```diff
@@ -156,8 +176,136 @@ func TestCartographerIntegration(t *testing.T) {
 
 	// Close out slam service
 	test.That(t, utils.TryClose(context.Background(), svc), test.ShouldBeNil)
+
```

Comment:
[minor] Add checks for new map file for the test cases where should be one. Also consider checking the timestamp of the map file to see if it aligns with map_rate.

Reply:
added a check for the offline case.